tardis/energy_input/tests/test_gamma_ray_transport.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+
+from tardis.model import SimulationState
+from tardis.io.configuration import config_reader
+from tardis.energy_input.gamma_ray_transport import (
+    calculate_shell_masses,
+    create_isotope_dicts,
+    get_all_isotopes,
+    create_inventories_dict,
+    calculate_total_decays,
+)
+import astropy.units as u
+import astropy.constants as c
+
+# pytest.fixtures.
+# Return the simulation object.
+
+
+@pytest.fixture()
+def config():
+    return config_reader.Configuration.from_yaml(
+        "/Users/anirbandutta/Projects/gamma_ray_tardis/tardis_configv1_density_exponential_nebular.yml"
+    )
+
+
+@pytest.fixture(scope="module", autouse=True)
+def simulation_setup(config):
+    config.model.structure.velocity.start = 1.0 * u.km / u.s
+    config.model.structure.density.rho_0 = 5.0e2 * u.g / (u.cm**3)
+    config.supernova.time_explosion = 1.0 * (u.d)
+    model = SimulationState.from_config(config)
+    return model
+
+
+def test_activity(simulation_setup):
+    """
+    ni_mass : mass of 56Ni in grams.
```

Comment:
docstrings should describe the function and input/output variables, not a variable in the middle of the function